Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -2206,7 +2217,8 @@ CGContextRef CGBitmapContextCreateWithData(void* data,
 
     ComPtr<ID2D1RenderTarget> renderTarget;
     RETURN_NULL_IF_FAILED(factory->CreateWicBitmapRenderTarget(customBitmap.Get(), D2D1::RenderTargetProperties(), &renderTarget));
-    return _CGBitmapContextCreateWithRenderTarget(renderTarget.Get(), image.get());
+    CGContextRef context = _CGBitmapContextCreateWithRenderTarget(renderTarget.Get(), image.get(), outputPixelFormat);
```

Comment:
There may be additional things here acting on context.

---
In reply to: [92289253](https://github.com/Microsoft/WinObjC/pull/1529#discussion_r92289253) [](ancestors = 92289253)